static/src/content/doc/getting-started.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Getting Started
+description: Getting started with Batteries Included.
+tags: ['getting-started', 'overview']
+draft: false
+---
+
+# Getting Started with Batteries Included
+
+Welcome to Batteries Included! We're excited to help you set up your modern
+infrastructure platform. Whether you're just exploring or ready to deploy
+production workloads, getting started is simple and straightforward.
+
+## What's Possible
+
+Batteries Included gives you everything needed to run modern applications:
+
+- **Databases**: Instantly deploy and manage PostgreSQL with automated backups
+  and monitoring.
+- **Web Services**: Handle web traffic with built-in load balancing and SSL.
+- **Monitoring**: Get insights with Grafana dashboards and VictoriaMetrics.
+- **Machine Learning**: Run AI workloads with vector databases and Jupyter
+  notebooks.
+- **Security**: Enterprise-grade security with OAuth and mTLS built-in.
```

Comment:
This is way too much wind up; Getting started is very few steps and we should be leaning into that.